packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -15,16 +27,21 @@ export interface LabelVariables {
 export default (siteVars: SiteVariablesPrepared): LabelVariables => {
   const color = 'rgba(0, 0, 0, 0.6)'
 
+  const colorScheme = extendColorScheme(siteVars.colorScheme, {
+    default: {
+      background: color,
+      foreground: 'rgb(232, 232, 232)',
+    },
+    brand: {
+      background: siteVars.colorScheme.brand.foreground4,
```

Comment:
not sure if this PR is going to address this, but as a client might see another issue with types. Here you might see that types suggest that `foreground` prop is guaranteed to be defined for each `color` of `colorSheme`. 

![image](https://user-images.githubusercontent.com/9024564/57776964-6b5afb80-7729-11e9-827d-f5fb05a4605a.png)

However, given this code, it is clear that value might be omitted - this is the case of `brand` color for Label's color scheme, where `foreground` is absent.

It would be great if our types will suggest the fact that value of area props (in our example, `foreground`) is, in fact, optional:

Reply:
The color scheme defined for the teams theme, already have all these values defined. In the labelVariables, we are just overriding some of those. Basically there is no need to override the brand foreground hover, that's why it is not defined.